### PR TITLE
Upgrade to upstream version

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *New version upstream*

## Solution
- *Upgrade to version upstream 2020-07-29*
- *Cleaning up READMEs*
- *Small typos all around*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
- [x] **Code review** : Kay0u
- [x] **Approval (LGTM)** :  Kay0u
*Code review and approval have to be from a member of @YunoHost-Apps/apps-group*
- **CI succeeded** :
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/dokuwiki_ynh%20PR66%20(ericgaspar)/)

### CODE REVIEW

**Review by kay0u — APPROVED** [review]
LGTM & code review

File: README_fr.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 
 DokuWiki est un logiciel wiki Open Source simple à utiliser et très polyvalent qui ne nécessite pas de base de données. Il est apprécié par les utilisateurs pour sa syntaxe propre et lisible. La facilité de maintenance, de sauvegarde et d'intégration en fait un favori d'administrateur. Des contrôles d'accès et des connecteurs d'authentification intégrés rendent DokuWiki particulièrement utile dans le contexte de l'entreprise et le grand nombre de plugins apportés par sa communauté dynamique permettent un large éventail de cas d'utilisation au-delà d'un wiki traditionnel.
 
-**Version incluse:** 2018-04-22b "Greebo"
+**Version incluse:** 2020-07-29
 
 ## Captures d'écran
 
-![Capture d'écran de la fenêtre principale de DokuWiki](dokuwikimainwindow.png)
+![Capture d'écran](https://www.dokuwiki.org/lib/exe/fetch.php?tok=341927&media=http%3A%2F%2Fimg832.imageshack.us%2Fimg832%2F8784%2Fdokuwikitemplatevector2.png)
```
Comment (Gofannon) [inline README_fr.md:19]: Might be better to have the picture locally than on the internet. It could be removed

File: conf/app.src [hunk 2]
```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://download.dokuwiki.org/src/dokuwiki/dokuwiki-2018-04-22b.tgz
-SOURCE_SUM=9d1437cdc7e98e471ff32079f7ca224ccf52dbbaafed39ec9746b0405b3a29ce
+SOURCE_URL=https://github.com/splitbrain/dokuwiki/archive/release_stable_2020-07-29.tar.gz
```
Comment (Gofannon) [inline conf/app.src:1]: Is there any reason for not sticking to the author website ?

**Review by Gofannon — COMMENTED** [review]
Thanks for your work!

I made non blocking remarks on the PR

File: conf/app.src (hunk 2 above)
Comment (ericgaspar) [reply conf/app.src:1]: I guess both work. it is just that GitHub repository gives info like changes log, update alert, issues... and it is perhaps less disturbing for someone who would like to do a PR DokuWiki upgrade.

Comment (Gofannon) [reply conf/app.src:1]: You're right. GitHub is more "appealing and widespread"

**Review by Gofannon — APPROVED** [review]
Good for me. Code read but not tested

**Review by kay0u — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # DokuWiki for YunoHost
 
-[![Integration level](https://dash.yunohost.org/integration/dokuwiki.svg)](https://dash.yunohost.org/appci/app/dokuwiki)  
+[![Niveau d'intégration](https://dash.yunohost.org/integration/dokuwiki.svg)](https://dash.yunohost.org/appci/app/dokuwiki) ![](https://ci-apps.yunohost.org/ci/badges/dokuwiki.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/dokuwiki.maintain.svg)
```
Comment (yalh76) [inline README.md:3]:
```suggestion
[![Integration level](https://dash.yunohost.org/integration/dokuwiki.svg)](https://dash.yunohost.org/appci/app/dokuwiki) ![](https://ci-apps.yunohost.org/ci/badges/dokuwiki.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/dokuwiki.maintain.svg)  
```

File: README.md [hunk 4]
```diff
@@ -12,11 +12,11 @@ If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to
 
 DokuWiki is a simple to use and highly versatile Open Source wiki software that doesn't require a database. It is loved by users for its clean and readable syntax. The ease of maintenance, backup and integration makes it an administrator's favorite. Built in access controls and authentication connectors make DokuWiki especially useful in the enterprise context and the large number of plugins contributed by its vibrant community allow for a broad range of use cases beyond a traditional wiki.
 
-**Shipped version:** 2018-04-22b "Greebo"
+**Shipped version:** 2020-07-29
 
 ## Screenshots
 
-![Screenshot of DokuWiki main window](dokuwikimainwindow.png)
+![Screenshot](sources/DokuWiki_Screenshot.png)
```
Comment (yalh76) [inline README.md:19]:
```suggestion
![Screenshot of DokuWiki main window](sources/DokuWiki_Screenshot.png)
```

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Review by yalh76 — APPROVED** [review]
Nice work ;)

**Comment by alexAubin** [comment]
Let's merge this since everybody seems happy with the PR ? :thinking:

**Pull request merged**